### DISCUSSION TITLE
Welcome to remix-express-vite-plugin Discussions!

### DISCUSSION THREAD

**Discussion by kiliman** [kiliman/remix-express-vite-plugin#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jansedlon** [comment]
Hi! I'm Jan Sedloň and I'm a co-founder of @flixy-dev. Right now we have about 300 paying customer but we grow fast (that's subjective). I've kept my eye on Remix since it has been a paid framework but started using Remix since v1.

**Reply by kiliman** [reply]
Hi Jan, glad to have you around. 👋
